app/controllers/ConfigurationController.scala

```diff
@@ -68,7 +67,8 @@ class ConfigurationController @Inject()(val messagesApi: MessagesApi) extends Co
   def updateDataSetDefault(dataSetName: String) = SecuredAction.async(parse.json(maxLength = 20480)) { implicit request =>
     for {
       dataset <- DataSetDAO.findOneBySourceName(dataSetName) ?~> Messages("dataset.notFound")
-      _ <- (request.identity.isAdminOf(dataset.owningOrganization) || request.identity.isTeamManagerInOrg(dataset.owningOrganization)) ?~> Messages("notAllowed")
+      organization <- OrganizationSQLDAO.findOneByName(dataset.owningOrganization)
+      _ <- Fox.assertBoolean(request.identity.isTeamManagerOrAdminOfOrg(organization._id)) ?~> Messages("notAllowed")
```

Comment:
Could we just use `ensureTeamAdministration` instead?

Reply:
Yes, will do.

Reply:
or not, since this is not team-specific (kind of a special case, all teammanagers are allowed to do this, no matter which team they are in). But I did this replacement in another instance.